test/qpu_interface.jl

```diff
@@ -1510,6 +1576,70 @@ end
     end
 end
 
+@testset "get_connectivity on AnyonYukonQPU and AnyonYamaskaQPU given excluded positions" begin
+
+    expected_yukon_connectivity = Snowflurry.with_excluded_connections(
+        Snowflurry.with_excluded_positions(Snowflurry.AnyonYukonConnectivity, [3, 4, 5, 6]),
+        [(4, 5), (5, 6)],
+    )
+
+    expected_yamaska_connectivity = Snowflurry.with_excluded_connections(
+        Snowflurry.with_excluded_positions(
+            Snowflurry.AnyonYamaskaConnectivity,
+            [7, 8, 9, 10, 11, 12],
+        ),
+        [(7, 12), (12, 8)],
+    )
+
+    test_specs = [
+        (
+            "yukon",
+            AnyonYukonQPU,
+            yukonMetadataWithExcludedComponents,
+            expected_yukon_connectivity,
+        ),
+        (
+            "yamaska",
+            AnyonYamaskaQPU,
+            yamaskaMetadataWithExcludedComponents,
+            expected_yamaska_connectivity,
+        ),
+    ]
+
+    expected_project_id = ""
+
+    function not_implemented_request_checker(
+        url::String,
+        user::String,
+        input_access_token::String,
+        body::String,
+        realm::String,
+    )
+        throw(ErrorException("request checker should not be called during test"))
+    end
+
+    for (qpu_name, QPU, metadata, expected_connectivity) in test_specs
+
+        test_client = Client(
+            host = expected_host,
+            user = expected_user,
+            access_token = expected_access_token,
+            requestor = requestor = MockRequestor(
```

Comment:
Typo (requestor = requestor = ...)

Reply:
goot catch! puzzled as to why it doesn't crash